config_defaults_inc.php

```diff
@@ -2939,6 +2939,23 @@
 $g_anonymous_account = '';
 
 /**
+ * Enable usage of public share URLs. The default setting is OFF.
+ * Enabling this option will show additional "Public Share URL" read-only field 
+ * when viewing the bug (in "Users monitoring this issue" section) as well as 
+ * additional "token" field in SOAP API. The URL can be copied, used and shared. 
+ * Access via public share URL does not require loggin in (e.g. access is anonymous).
+ * This option does not require $g_allow_anonymous_login to be enabled.
+ * When user visits public share URL - he will see a limited read-only view of the 
+ * bug with "Full View" button asking to log in. User may see current status of the 
+ * bug and minor details (such as modification date). Public bugs will show summary, 
+ * description and steps to reproduce in limited view. Private bugs will show only 
+ * summary, but no description and steps to reproduce. No other data nor actions are 
+ * available in the limited view for public URLs.
+ * @global integer $g_public_urls
+ */
+$g_public_urls = OFF;	
```

Comment:
I just using $g_feature_name_enabled.  This way, the intention of the config variable is clear, and as needed, we can add more related configs as $g_feature_name_xyz, etc.  I'm not sure "public_urls" is the best name.  Are they public URLs, private URLs, secret URLs, ??  Not sure.  Maybe we should check for naming in features like YouTube, Google Docs, and other tools that offer similar access patterns.